.circleci/config.yml

```diff
@@ -181,8 +181,8 @@ jobs:
           command: |
             echo ${NVM_DIR}
             [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
-            nvm install v10.15.3
-            nvm alias default v10.15.3
+            nvm install v12.18.4
```

Comment:
good you update it, but I think it should be also in sync with Dockerfile: https://github.com/mojaloop/auth-service/blob/master/Dockerfile#L1

Reply:
Ok I will update it to be `12.16.1` to be consistent with the docker image